packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain/SourceNetworkBox.tsx

```diff
@@ -162,17 +184,33 @@ export function SourceNetworkBox({
             onChange={e => setAmount(e.target.value)}
           />
 
-          {isBatchTransferSupported && (
-            <TransferPanelMainInput
-              maxButtonOnClick={amount2MaxButtonOnClick}
-              errorMessage={errorMessages?.inputAmount2}
-              value={amount2}
-              onChange={e => setAmount2(e.target.value)}
-              tokenButtonOptions={{
-                symbol: nativeCurrency.symbol,
-                disabled: true
-              }}
-            />
+          {isBatchTransferSupported && !isAmount2InputVisible && (
+            <div className="flex justify-end">
+              <Amount2ToggleButton
+                onClick={() => setIsAmount2InputVisible(true)}
+              />
+            </div>
+          )}
+
+          {isBatchTransferSupported && isAmount2InputVisible && (
+            <>
+              <TransferPanelMainInput
+                maxButtonOnClick={amount2MaxButtonOnClick}
+                inputCollapseOnClick={() => setIsAmount2InputVisible(false)}
+                errorMessage={errorMessages?.inputAmount2}
+                value={amount2}
+                onChange={e => setAmount2(e.target.value)}
+                tokenButtonOptions={{
+                  symbol: nativeCurrency.symbol,
+                  disabled: true
+                }}
+              />
+              <p className="mt-1 text-xs font-light text-white">
+                You are able to move ETH in the same transaction, but you are
+                not required to do so. This is the minimum ETH amount you will
+                get, but you may get a bit more dependent on the gas usage.
```

Comment:
```suggestion
                You can transfer ETH in the same transaction if you wish to. This is the approximate amount you will receive. The final amount depends on actual gas usage. It usually varies by 0.00x ETH.
```

we should give an approx of 0.00x ETH so that users know what to expect

i am not entirely sure if it is the minimum or can be less than the amount user inputs
feel free to adjust the text accordingly

Reply:
+1 - The content here should be approved by the product team.

Reply:
I'm not sure about specifying the approximate amount, it's pretty much the amount you write in the amount2 input